controllers/sr_controller/sr_controller.py

```diff
@@ -101,11 +101,30 @@ def get_robot_mode() -> str:
     return MODE_FILE.read_text().strip()
 
 
+def print_simulation_version() -> None:
+    version_path = (ROOT / '.simulation-rev')
+    if version_path.exists():
+        description, revision = version_path.read_text().splitlines()
+        version = "{} (rev {})".format(description, revision)
+    else:
+        version = subprocess.check_output(
+            ['git', 'describe', '--always', '--tags'],
+            cwd=str(ROOT.resolve()),
+        ).decode().strip()
+
+    print("Running simulator version {}".format(version))
+
+
 def main():
     robot_mode = get_robot_mode()
     robot_zone = get_robot_zone()
     robot_file = get_robot_file(robot_zone, robot_mode)
 
+    if robot_zone == 0:
```

Comment:
Not sure how I feel about this. It's definitely possible for Zone 0 to be unpopulated (however unlikely), and given the  below `print`, it's not actually reducing any noise.

Reply:
This print doesn't convey any more information if it appears four times though, the other one conveys different information per zone.

Bear in mind also that it seems somewhat unlikely that a team will modify their simulation world to remove the zone 0 controller. They're much more likely to move their own code around, which doesn't affect this (as it runs outside user-code space).

Since this is primarily for _our_ convenience I think the noise reduction here is worth it.

Reply:
Good point, good enough for me!